assembl/tests/views_tests/test_graphql.py

```diff
@@ -1455,25 +1455,47 @@ def test_query_default_discussion_preferences(graphql_request,
     }
 
 
-def test_mutation_language_preference(graphql_request,
-                                      discussion_with_lang_prefs):
+def test_mutation_update_language_preference(graphql_request,
+                                             discussion_with_lang_prefs):
     res = schema.execute(u"""
 mutation myMutation($languages: [String]!) {
-    updateDiscussionPreference($languages) {
-        languages {
-            locale
+    updateDiscussionPreference(languages: $languages) {
+        preferences {
+            languages {
+                locale
+            }
         }
     }
 }
 """, context_value=graphql_request,
         variable_values={
-            "languages": ["ja, de"]
+            "languages": ["ja", "de"]
```

Comment:
wow, I would not have committed my test until it does work, but that's a way of doing it

Reply:
Haha I'll note this for the future.